phpBB/styles/prosilver/theme/mentions.css

```diff
@@ -0,0 +1,87 @@
+/* -------------------------------------------------------------- /*
+	$Mentions
+/* -------------------------------------------------------------- */
+
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable selector-no-qualifying-type */
+
+/* Mention block
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+/* Mention dropdown
+---------------------------------------- */
+.atwho-view { /* mention-container */
+	text-align: left;
+	border-radius: 2px;
+	position: absolute;
+	z-index: 999;
+	top: 20px;
+	overflow: auto;
+	transition: all 0.2s ease;
+}
+
+.atwho-view-ul { /* mention-list */
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.mention-media {
+	display: inline-flex;
+	flex-shrink: 0;
+	justify-content: center;
+	align-items: center;
+	margin-right: 10px;
+	margin-left: 0;
+}
+
+.mention-media-avatar {
+	font-size: 32px;
+	line-height: 36px;
+	text-align: center;
+	vertical-align: center;
+	width: 36px;
```

Comment:
you should not need anything here but the width and height which should be 40px. Use svgs for the icons, not font-awsome...Were switching to svgs in 3.3 anyway...